### PR TITLE
Performance tests

### PR DESCRIPTION
**Pull request by victuxbb**
# Background

We want to know how much is performance affected by all kind of ads.

[PSP-2046](https://jira.scmspain.com/browse/PSP-2046)

# Goal

Have all possible scenarios in a local http server to be tested with LightHouse tool from google.

# Implementation

* Add performance task to build all possible cases
* Add initial js logic for every case
* Add performance base template for all cases
* Add Webpack configuration for performance cases

# Further considerations

Tests are dependant on the performance of the owner who run the tests. We will do an average of every case 3 times.

# Checklist

- [x] The PR relates to *only* one subject with a clear title.
- [ ] I have performed a self-review of my own code
- [x] Wrote [good commit messages](http://chris.beams.io/posts/git-commit/)
- [x] My code is readable by someone else, and I commented the hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Checklist
+
+- [ ] The PR relates to *only* one subject with a clear title.
+- [ ] I have performed a self-review of my own code
+- [ ] Wrote [good commit messages](http://chris.beams.io/posts/git-commit/)
+- [ ] My code is readable by someone else, and I commented the hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+
```
Comment (alextremp) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion

# Visual Description

*put your meme here*
```

Comment (alextremp) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion

# Visual Description

*put your meme here*
```

File: src/ui/css/demo.css [hunk 2]
```diff
@@ -44,8 +44,8 @@ p {
 }
 
 .head {
-    background: url(../img/img-head.jpg) center center no-repeat;
-    background-size: cover;
+    /*background: url(../img/img-head.jpg) center center no-repeat;*/
```
Comment (alextremp) [inline src/ui/css/demo.css]: can it be removed?

File: src/ui/css/demo.css [hunk 3]
```diff
@@ -193,8 +194,8 @@ code {
 }
 
 .foot {
-    background: url(../img/img-foot.jpg) center center no-repeat;
-    background-size: cover;
+    /*background: url(../img/img-foot.jpg) center center no-repeat;*/
```
Comment (alextremp) [inline src/ui/css/demo.css]: also this, can it be removed?

**Review by alextremp — APPROVED** [review]
👍 

It looks fine to me, but please check the recommendations to update it they look good to you :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (alextremp) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion

# Visual Description

*put your meme here*
```

Comment (alextremp) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion

# Visual Description

*put your meme here*
```

Comment (alextremp) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion

# Visual Description

*put your meme here*
```

**Review by rafamarquezv — APPROVED** [review]

File: src/ui/performance_template.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            </div>
+            <div id="single_video"></div>
+            <div class="bubbles">
+                <div id="bubble1"></div>
+                <div id="bubble2"></div>
+                <div id="bubble3"></div>
+            </div>
+            <div id="single_native"></div>
+        </div>
+
```
Comment (pablovegau) [inline src/ui/performance_template.html]: There are multiple blank lines between html tags.

File: src/ui/performance_template.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            </div>
+
+            <div id="banner_bubbles">
+                <div id="banner_bubbles1"></div>
+
+            </div>
+
+        </div>
+
+
```
Comment (pablovegau) [inline src/ui/performance_template.html]: There are multiple empty blank lines between html tags.

**Review by pablovegau — APPROVED** [review]

**Pull request merged**